File: c/triangle_checker.c
#include <stdio.h>

int main()
{
    int side_one,side_two,side_three;
    printf("Tell me all 3 side length of triangle\n");
    printf("First Side\n");
    scanf("%d",&side_one);
    printf("Second Side\n");
    scanf("%d",&side_two);
    printf("Third Side\n");
    scanf("%d",&side_three);
    if (side_one + side_two >= side_three || side_two + side_three >= side_one || side_one + side_three >= side_two)
    {
        if (side_one == side_two && side_one == side_three)
        {
            printf("It is an Equilateral Triangle!");
        }
        else if (side_one == side_two || side_one == side_three || side_three == side_two)
        {
            printf("It is an Isosceles Triangle!");
        }
        else
        {
            printf("It is a Scalene Triangle!");
        }
    }
    else
    {
        printf("Its not a Valid Triangle!!");
    }
}

File: python/triangle_checker.py
side_one=int(input("Tell me first side size: "))
side_two=int(input("Tell me second side size: "))
side_three=int(input("Tell me third side size: "))

if side_one+side_two>=side_three or side_one+side_three>=side_two or side_two+side_three>=side_one:
    if side_one == side_two == side_three:
        print("It's an Equilateral Triangle!")
    elif side_one == side_two or side_two == side_three or side_one == side_three:
        print("Its an Isosceles Triangle!")
    else:
        print("Its a Scalene Triangle")
else:
    print("Not a valid Triangle")